i18n/en-US.properties

```diff
@@ -373,7 +373,9 @@ be.nextFile = Next File
 # Title for next segment on skill timeline
 be.nextSegment = Next Segment
 # Message displayed in an empty activity feed
-be.noActivity = No Activity Yet
+be.noActivity = No activity to show
+# Message shown alongside empty activity feed when user can annotate
+be.noActivityAnnotationPrompt = Hover over the preview and use the control at the bottom to annotate the file.
```

Comment:
Should `control` be `controls`?

Reply:
Good call, we'll be adding more in the future